src/sentry/projectoptions/defaults.py

```diff
@@ -14,7 +14,7 @@
 # epoch instead.
 LEGACY_GROUPING_CONFIG = "legacy:2019-03-12"
 DEFAULT_GROUPING_CONFIG = "newstyle:2019-10-29"
-BETA_GROUPING_CONFIG = "mobile:2021-02-12"
+BETA_GROUPING_CONFIG = "beta:2022-12-20"
```

Comment:
I would already name this `newstyle:2022-12-20` so we can just make it stable with that name when we release it.